instrumentation/dalli/lib/opentelemetry/instrumentation/dalli/utils.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+# Copyright 2020 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    # Utility functions
+    module Utils
+      extend self
+
+      STRING_PLACEHOLDER = ''.encode(::Encoding::UTF_8).freeze
+      CMD_MAX_LEN = 500
+
+      OPNAME_MAPPING = {
+        'get'           => 'get',
+        'cas'           => 'get',
+        'set'           => 'set',
+        'add'           => 'add',
+        'replace'       => 'replace',
+        'delete'        => 'delete',
+        'incr'          => 'incr',
+        'decr'          => 'decr',
+        'flush'         => 'flush',
+        'write_noop'    => 'noop',
+        'version'       => 'version',
+        'send_multiget' => 'getkq',
+        'append'        => 'append',
+        'prepend'       => 'prepend',
+        'stats'         => 'stat',
+        'reset_stats'   => 'stat',
+        'multi_set'     => 'setq',
+        'multi_add'     => 'addq',
+        'multi_replace' => 'replaceq',
+        'multi_delete'  => 'deleteq',
+        'touch'         => 'touch',
+        # 'sasl_authentication' => 'auth_negotiation',
+        # 'sasl_authentication' => 'auth_request',
+      }.freeze
+
+      def opname(operation, multi)
+        lookup_name = multi ? "multi_#{operation}" : operation.to_s
```

Comment:
I don't like [repeating the multi check](https://github.com/petergoldstein/dalli/blob/master/lib/dalli/server.rb#L289) for getting the op code, but I didn't think of a better way of doing it.  Open to suggestions for improvement here.

Reply:
It's a pretty trivial boolean check - I think this is fine.